test/integration/shim_int_test.go

```diff
@@ -31,11 +32,14 @@ func (suite *ShimIntegrationTestSuite) TestShim_Basic() {
 print("Hello World!")
 	`
 
-	testScript := filepath.Join(fmt.Sprintf("%s/%s.pl", ts.Dirs.Work, suite.T().Name()))
+	testScript := filepath.Join(fmt.Sprintf("%s/%s.py", ts.Dirs.Work, suite.T().Name()))
 	fail := fileutils.WriteFile(testScript, []byte(scriptBlock))
 	suite.Require().NoError(fail.ToError())
 
-	cp := ts.Spawn("shim", "--", "python3", fmt.Sprintf("%s", testScript))
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("shim", "--", "python3", fmt.Sprintf("%s", testScript)),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
```

Comment:
Wonder if we can make these tests a bit cheaper by disabling runtime and just asserting that the environment was sourced some other way (eg. just check `ACTIVESTATE_ACTIVATED_ID` or something -- and ensure its value is different from the one that was passed to the current process that's running the tests)